DataTables.Blazor/Interop/DatatablesInterop.cs

```diff
@@ -68,7 +68,12 @@ public DataTablesInterop(IJSRuntime runtime)
 
         /// <inheritdoc/>
         public ValueTask InitialiseAsync(ElementReference tableReference, DataTableOptions options)
-            => _runtime.InvokeVoidAsync("datatablesInterop.initialiseDataTable", tableReference, JsonSerializer.Serialize(options, SerializerOptions));
+        {
+            var serializedOptions = JsonSerializer.Serialize(options, SerializerOptions);
```

Comment:
Was this just for debugging?

Reply:
Yes - it helped debugging the DataTables.js (annoying) implementation of buttons. Release compilation will remove the extra allocation so I figure it's OK for it to stay

Reply:
That's cool.